scripts/setup.sh

```
@@ -91,6 +92,10 @@ if is_checked dotfiles; then
   print_step 'Setting up zsh' &&
     source "$DOTFILES_DIR/scripts/zsh.sh"
 
+  # These land in ~/.claude and ~/.codex and steer every agent session on the
+  # machine, so they are opt-in rather than part of the dotfiles bundle.
+  is_checked agents || STOW_SKIP_AGENTS=true
```

Shellcheck (0.11.0)

[warning] STOW_SKIP_AGENTS appears unused. Verify use (or export if used externally).

(SC2034)